apps/ledger-live-desktop/src/renderer/components/CustomImage/CustomImageDeviceAction.tsx

```diff
@@ -83,7 +91,8 @@ const CustomImageDeviceAction: React.FC<Props> = withRemountableWrapper(props =>
     if (error instanceof ImageCommitRefusedOnDevice) {
       dispatch(clearLastSeenCustomImage());
     }
-  }, [dispatch, error]);
+    onError && onError(error);
```

Comment:
You should check here that `error` is not undefiend or null before calling `onError` no ?